### PR TITLE
modify kip-1: from klaytn to kaia

### PR DESCRIPTION
**Pull request by praveen-kaia**
## Proposed changes

- Kaia Changes

## Types of changes

Please put an x in the boxes related to your change.

- [X] Bugfix
- [ ] KIP Proposal
- [ ] KIP Improvement

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [X] Used the suggested template: https://github.com/kaiachain/KIPs/blob/main/kip-template.md
- [X] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribution
- [X] I have added necessary documentation (if appropriate)
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# GitHub highlighting for Solidity files
+# See https://github.com/github/linguist/pull/3973#issuecomment-357507741
+*.sol linguist-language=Solidity
```
Comment (yoomee1313) [inline .gitattributes:3]: Is this file needed? It's about solidity.

File: KIPs/kip-1.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ Following is the process that a successful KIP will move along:
 
 Each status change is requested by the KIP author and reviewed by the KIP editors. Use a pull request to update the status. Please include a link to where people should continue discussing your KIP. The KIP editors will process these requests as per the conditions below.
 
-* **Idea** -- Once the champion has asked the Klaytn community whether an idea has any chance of support, they will write a draft KIP as a [pull request](https://github.com/klaytn/KIPs/pulls). Consider including an implementation if this will aid people in studying the KIP.
+* **Idea** -- Once the champion has asked the Kaia community in  https://devforum.kaia.io/c/kips whether an idea has any chance of support, they will write a draft KIP as a [pull request](https://github.com/kaiachain/KIPs/pulls). Consider including an implementation if this will aid people in studying the KIP.
```
Comment (yoomee1313) [inline KIPs/kip-1.md:72]: Can you add `TODO` or `TBD` around here? Since the link will be changed.

File: _config.yml [hunk 3]
```diff
@@ -23,6 +23,10 @@ twitter_username: kaiachain
 github_username: kaiachain
 facebook_username: kaiachain
 medium_username: kaiachain
+repository: kaiachain/KIPS
```
Comment (yoomee1313) [inline _config.yml:26]: `KIPS`? Can I ask what this link is?

File: README.md [hunk 4]
```diff
@@ -1,29 +1,62 @@
...
+- [KCTs](https://kips.kaia.io/token) specify the various token standards in Kaia.
+- [SDKs](https://kips.kaia.io/sdk) specify software developer kits and its architecture to support Kaia.
+- [Application](https://kips.kaia.io/application) specify application layer standards, which determine how applications running on Kaia can interact with each other.
+- [Meta KIPs](https://kips.kaia.io/meta) are miscellaneous improvements that nonetheless require some sort of consensus.
+- [Informational KIPs](https://kips.kaia.io/informational) are non-standard improvements that do not require any form of consensus.
 
-If your KIP requires images, the image files should be included in a subdirectory of the `assets` folder for that KIP as follows: `assets/kip-N` (where **N** is to be replaced with the KIP number). When linking to an image in the KIP, use relative links such as `../assets/kip-1/image.png`.
+**Before you write an KIP, ideas MUST be thoroughly discussed on [Kaia Research](https://devforum.kaia.io/c/kips). Once consensus is reached, thoroughly read and review [KIP-1](https://kips.kaia.io/KIPS/kip-1), which describes the KIP process.**
```
Comment (ian0371) [inline README.md:16]:
```suggestion
**Before you write a KIP, ideas MUST be thoroughly discussed on [Kaia Research](https://devforum.kaia.io/c/kips). Once consensus is reached, thoroughly read and review [KIP-1](https://kips.kaia.io/KIPS/kip-1), which describes the KIP process.**
```

please also fix others as well

File: _config.yml (hunk 3 above)
Comment (praveen-kaia) [reply _config.yml:26]: I followed https://github.com/ethereum/EIPs/blob/master/_config.yml , but yeah i dont see this term in the jekyll library. will remove this.

File: README.md (hunk 4 above)
Comment (praveen-kaia) [reply README.md]: Added TBD for forum

File: _config.yml (hunk 3 above)
Comment (praveen-kaia) [reply _config.yml]: Removed this

File: KIPs/kip-1.md (hunk 2 above)
Comment (praveen-kaia) [reply KIPs/kip-1.md]: Added TBD for forum link

File: .gitattributes (hunk 1 above)
Comment (praveen-kaia) [reply .gitattributes]: removed the file

**Review by yoomee1313 — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (ian0371) [reply README.md]: @praveen-kaia  Do you mean you'll change the articles (an -> a) in the future?

Comment (praveen-kaia) [reply README.md]: Added suggested change @ian0371 .

Comment (ian0371) [reply README.md]: could you also change two more occurrences?

Comment (praveen-kaia) [reply README.md]: @ian0371 , i don't see any other occurrences of this change. can you check once.

**Review by ian0371 — APPROVED** [review]
